packages/components/src/DateTimePickers/MonthPicker/MonthPicker.component.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import theme from './MonthPicker.scss';
+import { chunk } from 'lodash';
+
+function MonthPicker(props) {
+	const months = [
+		'Jan',
+		'Fev',
+		'Mars',
+		'Avr',
+		'Mai',
+		'Juin',
+		'Juil',
+		'Aout',
+		'Sept',
+		'Oct',
+		'Nov',
+		'Dec',
+	];
+
+	const monthsRows = chunk(months, 3);
+
+	return (
+		<div className={theme.container}>
```

Comment:
Can you use a `<table>` ?
And like that, borders will be easier to manage. (Use `border-collapse` to avoid double borders)

Reply:
I was thinking of table but the semantic of multiple objects of same type was not here for me.
And i wanted to have only inner border so in both cases, the css is precise.